src/main/java/net/dv8tion/jda/core/entities/Guild.java

```diff
@@ -860,9 +860,9 @@ default Emote getEmoteById(long id)
     SnowflakeCacheView<Emote> getEmoteCache();
 
     /**
-     * Gets an unmodifiable list of the currently banned {@link net.dv8tion.jda.core.entities.User Users}.
-     * <br>If you wish to ban or unban a user, please {@link GuildController#ban(User, int) GuildController.ban(User, int)} or
-     * {@link GuildController#unban(User) GuildController.ban(User)}.
+     * Retrieves an unmodifiable list of the currently banned {@link net.dv8tion.jda.core.entities.User Users}.
+     * <br>If you wish to ban or unban a user, either please use {@link GuildController#ban(User, int) GuildController.ban(User, int)} or
```

Comment:
please use either

Reply:
might just drop the "please" too... cuz its not really a request... they can't ban/unban with getBanById so its a redirection